plugins/woocommerce/tests/e2e-pw/global-setup.js

```diff
@@ -118,20 +134,22 @@ module.exports = async ( config ) => {
 	for ( let i = 0; i < customerRetries; i++ ) {
 		try {
 			console.log( 'Trying to log-in as customer...' );
-			await customerPage.goto( `${ baseURL }/wp-admin` );
-			await customerPage.fill( 'input[name="log"]', 'customer' );
-			await customerPage.fill( 'input[name="pwd"]', 'password' );
+			await customerPage.goto( `/wp-admin` );
+			await customerPage.fill( 'input[name="log"]', customerUsername );
+			await customerPage.fill( 'input[name="pwd"]', customerPassword );
 			await customerPage.click( 'text=Log In' );
 
-			await customerPage.goto( `${ baseURL }/my-account/` );
+			await customerPage.goto( `/my-account` );
 			await expect(
-				customerPage.locator( 'h1.entry-title' )
-			).toContainText( 'My account' );
+				customerPage.locator(
+					'.woocommerce-MyAccount-navigation-link--customer-logout'
+				)
+			).toBeVisible();
```

Comment:
I had to change the assertion here, because the theme in our nightly smoke test site doesn't have a `h1.entry-title`. I thought, maybe if we assert that the logout link is visible, then the login was indeed successful.